ledger/README.md

```diff
@@ -61,6 +61,10 @@ interface:
   for recent blocks, if the tracker's state depends only on recent blocks
```

Comment:
I think we should a comment to loadFromDisk's description in the ledgerForTracker interface regarding eternal data.